src/handlers/comment/handlers/multiplier.ts

```diff
@@ -18,6 +19,11 @@ export const multiplier = async (body: string) => {
     return;
   }
 
+  if (!organization?.id) {
+    logger.info(`Skipping '/multiplier' because there's no organization details`);
+    return;
```

Comment:
Pls make sure that the bot returns a proper error message, there should not be cases when a bounty hunter executes a command and gets a silent response with no action

Reply:
> Pls make sure that the bot returns a proper error messages for all commands where `organization.id` is required, there should not be cases when a bounty hunter executes a command and gets a silent response with no action

What about I just use `repo id` if `org id` is not available?

Reply:
> Pls make sure that the bot returns a proper error messages for all commands where `organization.id` is required, there should not be cases when a bounty hunter executes a command and gets a silent response with no action

What about I just use `repo id` if `org id` is not available? @rndquu

Reply:
But we need organization id [later in the code](https://github.com/ubiquity/ubiquibot/pull/557/files#diff-6bcca3a69bf60fc9fbb11f0b546fe3dea32062d9b5ea8672f710e06338bc6d85R77) so, as far as I understand, using repository id doesn't really make sense

Reply:
> But we need organization id [later in the code](https://github.com/ubiquity/ubiquibot/pull/557/files#diff-6bcca3a69bf60fc9fbb11f0b546fe3dea32062d9b5ea8672f710e06338bc6d85R77) so, as far as I understand, using repository id doesn't really make sense

We save with either org or repo and read with same id as well.. depending on what is available

If that does not work then this will end up as an organization-only bot and bounty hunters should create organizations to test

Reply:
If we use `user_id_repository_id` then multipliers are going to be repository specific so bounty hunters may end up having different multipliers for different repositories in a single organization, right?

Reply:
no, its gonna be like `const id = organization?.id || repository?.id`

so, its prioritizing organizations but falling back to repository for bounty hunters test

Reply:
I think it's better to keep the code consistent, I mean using `organization.id` for all cases

The original change request is regarding the error messages, so pls update them accordingly

Reply:
> I think it's better to keep the code consistent, I mean using `organization.id` for all cases
> 
> The original change request is regarding the error messages, so pls update them accordingly

Done